packages/react-youtube/src/YouTube.tsx

```diff
@@ -93,6 +94,10 @@ export type YouTubeProps = {
    * Custom class name for the player container element
    */
   containerClassName?: string;
+  /**
+   * Custom class name for an iframe element
+   */
+  frameClassName?: string;
   /**
    * Custom styles for the player container element
    */
```

Comment:
Please change the prop name below to just `style`.

Reply:
Changed in 51b31f1.